phoenix-postgres-pgadmin/example_app/Dockerfile

```diff
@@ -0,0 +1,65 @@
+
+# ---- Build Stage ----
+FROM elixir:1.9-alpine AS app_builder
+
+# Set environment variables for building the application
+ENV MIX_ENV=prod
+ENV LANG=C.UTF-8
+
+# install build dependencies
+RUN apk add --update git build-base nodejs npm yarn python
+
+# Install hex and rebar
+RUN mix local.hex --force && \
+    mix local.rebar --force
+
+# Create the application build directory
+RUN mkdir /app
```

Comment:
No need to create manually the directory, the `WORKDIR` command will do that for you and won't create an extra layer as the `RUN` command will.
https://docs.docker.com/engine/reference/builder/#workdir

Reply:
removed